lists/th/th-tin.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": {
+    "en": "Thai Revenue Department - Tax Identification Number",
+    "local": "หมายเลขประจำตัวผู้เสียภาษี"
+  },
+  "url": "https://www.rd.go.th/publish/272.0.html",
+  "description": {
+    "en": "This is the standard tax number issued to legal entities that pay taxes or that pay income to employees. NGOs require one, too. The procedure is here: http://www.rd.go.th/publish/21987.0.html The government site still mentions a 10-digit number but the law changed in 2012 and the TIN is now 13 digits long (see: https://msnagroup.com/new-thai-tax-id-number). The TIN should not be confused with the PIN, for individuals."
```

Comment:
Individuals can be issued with a TIN if they are a foreigner or if they want to register for Value Added Tax or Specific Business Tax, according to http://www.rd.go.th/publish/21987.0.html. 

Also, foreign companies (and NGOs) operating in Thailand require a TIN and so possession of a TIN does not necessarily equate to an entity being on the Thai company register. That's my reading of it, anyway.

Are these points worth adding?

Reply:
I've edited to include those bits.

Reply:
Thanks @kd-ods - this looks good to merge now